client/web/src/site-admin/webhooks/WebhookLogPage.story.tsx

```diff
@@ -104,10 +116,10 @@ const buildWebhookLogs = (count: number, externalServiceCount: number): WebhookL
     return logs
 }
 
-export const NoLogs: Story = () => (
+export const NoLogs: Story = args => (
     <WebStory>
         {props => (
-            <MockedTestProvider mocks={buildHeaderMock(2, 2)}>
+            <MockedTestProvider mocks={buildHeaderMock(2, 2, args)}>
```

Comment:
I think this might actually want to be something like this, and use these two values in the build function:

```suggestion
            <MockedTestProvider mocks={buildHeaderMock(args.externalServiceCount, args. erroredWebhookCount)}>
```